src/popup/components/Modals/TransferReceiveBase.vue

```diff
@@ -132,6 +133,8 @@ import AccountItem from '../AccountItem.vue';
 
 import ShareIcon from '../../../icons/share.svg?vue-component';
 
+const SPLIT_PROTOCOL_ADDRESS = [PROTOCOL_BITCOIN, PROTOCOL_ETHEREUM];
```

Comment:
I suggest to add some comment on why those protocols have their addresses split.